modules/nodes-nodes-configuring-graceful-shutdown.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assembly:
+// * nodes/nodes-nodes-graceful-shutdown
+
+:_content-type: PROCEDURE
+[id="nodes-nodes-activating-graceful-shutdown_{context}"]
+= Configuring graceful node shutdown
+
+To configure graceful node shutdown for a node, create a `KubeletConfig` custom resource (CR) to specify a grace period and shutdown profile for the nodes corresponding to a specific `MachineConfigPool`. If graceful node shutdown is not configured, the default grace period is 0 and the pod is forcefully terminated before `preStop` has run to completion.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Define a `KubeletConfig` CR with graceful node shutdown.
+. Create the `KubeletConfig` object:
++
+[source,terminal]
+----
+$ oc create -f <filename>
+----
+
+. Example KubeletConfig
+Apply the following settings to configure graceful node shutdown.
+
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: KubeletConfig
+metadata:
+  name: graceful-shutdown
+  namespace: openshift-machine-config-operator
+spec:
+  machineConfigPoolSelector:
+    matchLabels:
+      pools.operator.machineconfiguration.openshift.io/worker: "" <1>
+  kubeletConfig:
+    shutdownGracePeriod: "3m" <2>
+    shutdownGracePeriodCriticalPods: "2m" <3>
+EOF
+----
+<1> Apply graceful node shutdown to all worker nodes.
+<2> Define a shutdown period of 3 minutes for regular pods.
+<3> Define a grace period for critical pods of 2 minutes. By default, `shutdownGracePeriod` and `shutdownGracePeriodCriticalPods` are set to 0.
+
+You can create and apply a custom `KubeletConfig` to configure grace periods and associated pods.
+Set the `shutdownGracePeriod` and `shutdownGracePeriodCriticalPods` timings  for normal and critical pods.
+Label one or more `MachineConfigPools` to apply the custom kubelet configuration to one or more nodes in a pool.
+
+In the following example, kubelet delays node shutdown for 30 minutes. 20 minutes are reserved for normal pods. 10 minutes are reserved for "critical" pods (pods with priority >=2000000000).
```

Comment:
Just noting I still see this example in modules/nodes-nodes-grace-periods-graceful-shutdown.adoc. Should the example be in both places?

Reply:
Removed.